bin/crew

```diff
@@ -991,7 +989,8 @@ def shrink_dir(dir)
         # Logic here is to find executable binaries, compress after making
         # a backup, then expand the compressed file with upx. If the
         # expansion doesn't error out then it is ok to delete the backup.
-        @execfiles = %x[find . -executable -type f ! \\( -name \"*.so*\" -o -name \"*.a\" \\) -exec sh -c \"file -i \'{}\' | grep -q \'executable; charset=binary\'\" \\; -print].chomp
+        # Disable for sommelier.elf and Xwayland.elf since upx breaks those binaries.
```

Comment:
Are we sure these are the only binaries that break with upx?  I guess we'll find out on the next few builds.  This means we need to test every binary for package updates.  We should probably do that anyway but this will make testing mandatory.

Reply:
I've been doing updates with shrink enabled for many weeks. No complaints so far...

Reply:
(of course I tend to run the apps too...)